src/services/member.ts

```diff
@@ -116,3 +116,16 @@ export async function isCurrentMemberVerified(): Promise<boolean> {
     )
     .catch(() => false);
 }
+
+export const getAllAdmins = async (): Promise<Member[]> => {
+  const url = `/members/admin/`;
+
+  const res = await api.get(url);
+
+  devPrint('res', res);
```

Comment:
nit: let's try to either avoid adding logs or add a more descriptive name going forward, just so the console isn't crowded by a bunch of `res: <>` logs during development

Reply:
Got it. I got rid of the ones I recently added.